WebDriverAgentLib/Routing/FBElementUtils.m

```diff
@@ -12,16 +12,39 @@
 #import "FBElementUtils.h"
 #import "FBElementTypeTransformer.h"
 
+NSString *const FBUnknownAttributeException = @"FBUnknownAttributeException";
 static NSString *const WD_PREFIX = @"wd";
 
 @implementation FBElementUtils
 
 + (NSString *)wdAttributeNameForAttributeName:(NSString *)name
```

Comment:
This method is heavily used. And you are recalculating everything every single time.